dart/lib/src/sentry_options.dart

```diff
@@ -139,7 +139,7 @@ class SentryOptions {
 
   /// Sentry client name used for the HTTP authHeader and userAgent eg
   /// sentry.{language}.{platform}/{version} eg sentry.java.android/2.0.0 would be a valid case
-  String? sentryClientName;
+  late String sentryClientName;
```

Comment:
Maybe this could be a `get` that returns the data from the `sdk`?
So we don't need to make it `late`.

https://github.com/getsentry/sentry-dart/pull/1133#discussion_r1022537003

Reply:
Should this value not be mutable by the user but rather readonly?

Reply:
It is mutable, via the `sdk` object, this one just quickly gives you the right format `$name/$version`.